README_JP4.md

```diff
@@ -0,0 +1,133 @@
+# Intel® RealSense™ camera driver for GMSL* interface on Jetson AGX Xavier JetPack 4.6.1
+# D457 MIPI on Jetson AGX Xavier
+The RealSense MIPI platform driver enables the user to control and stream RealSense 3D MIPI cameras.
+
+The system shall include:
+* Jetson platform (Currently Supported JetPack versions are: 4.6.1)
+* RealSense De-Serialize board (https://store.intelrealsense.com/buy-intel-realsense-des457.html)
+* RS MIPI camera (e.g. https://store.intelrealsense.com/buy-intel-realsense-depth-camera-d457.html)
+
+> Note: This MIPI reference driver is based on RealSense de-serialize board. For other de-serialize boards, modification might be needed.
+
+# Links
+
+- Jetson AGX Orin board setup - AGX Orin [JetPack 6.0](./README_JP6.md) setup guide
```

Comment:
Why do we nned all those links?
The parent had it.
It also has a link to itself

Reply:
For better navigation